plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1083,10 +1095,13 @@ def main():
         try:
             vm = get_vm(proxmox, vmid)
             if not vm:
-                module.exit_json(changed=False, msg="VM %s does not exist" % vmid)
```

Comment:
Could I ask why this `msg` has been removed?

Reply:
Only commit 78d8c03ff871634a194eab1655055ae053cd0be1 is relevant for this particular PR. This particular modification is part of PR #811 will be gone if it gets merged.

Reply:
I've added a`[WIP]` in the PR title to make it clear that it is not done and waits for something (in this case, #811).

Reply:
Rebased on main branch.